torch/nn/_intrinsic/quantized/modules/conv_relu.py

```diff
@@ -22,6 +24,18 @@ def __init__(self, in_channels, out_channels, kernel_size, stride=1,
                                          stride=stride, padding=padding, dilation=dilation,
                                          groups=groups, bias=bias, padding_mode=padding_mode)
 
+    @property
+    def weight(self):
```

Comment:
Did we decide to keep properties for the weight?

Reply:
no, these should be removed

Reply:
this pr already has too many changes, will do it in a separate pr

Reply:
If you are inheriting from nn.quantized.conv2d, with the latest changes, you dont need these functions anymore, delete these lines?